pennylane/transforms/control.py

```diff
@@ -127,7 +127,7 @@ def adjoint(self):
             # Not within a queuing context
             with QuantumTape() as new_tape:
                 # Execute all ops adjointed.
-                ops = adjoint(requeue_ops_in_tape)(self.subtape)
+                ops = adjoint(requeue_ops_in_tape)(self.subtape)  # is this needed?

```

Comment:
@KetpuntoG It seems that it is not used anywhere. We can silent the `unused-variable` warning though.

Reply:
It seems that codefactor does not fail even with the issue being detected. But it fails elsewhere, e.g., [here](https://github.com/PennyLaneAI/pennylane/pull/1966).